src/vanilla/Model/CodeModelJv.cs

```diff
@@ -35,6 +35,19 @@ public override string BaseUrl
         [JsonIgnore]
         public bool IsCustomBaseUri => Extensions.ContainsKey(SwaggerExtensions.ParameterizedHostExtension);
 
+        private bool? shouldGenerateXmlSerializationCached = null;
+        public bool ShouldGenerateXmlSerializationCached

```

Comment:
Having a Cached and "NonCached" version feels a little funky to me. Is there a way to merge them into a single property?

Reply:
It's declared in the core project and they're not comfortable with making it lazy because of mutability.